sources/packages/web/src/types/contracts/RestrictionContract.ts

```diff
@@ -86,3 +86,15 @@ export enum RestrictionNotificationType {
    */
   Error = "Error",
 }
+
+// Student restriction badge.
+export enum StudentRestrictionStatus {
+  noRestriction = "No restrictions",
```

Comment:
I am assuming that this was an existing enum that still needs to be changed to the most updated naming convention, right?

Reply:
yes, it is. I will update